src/test/java/core/basesyntax/service/CsvFileReaderServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.AcceptedDataInvalidException;
+import core.basesyntax.service.impl.CsvFileReaderServiceImpl;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CsvFileReaderServiceImplTest {
+    private static CsvFileReaderService fileReaderService;
+
+    @BeforeAll
+    static void setUp() {
+        fileReaderService = new CsvFileReaderServiceImpl();
+    }
+
+    @Test
+    void readLines_readFromEmptyFile_ok() {
+        String filePath = "src/main/java/core/basesyntax/resources/Empty.csv";
```

Comment:
replace the filePath variable with a constant (in other places where you use the path, too)